app/models/Application.scala

```diff
@@ -91,19 +93,19 @@ case class Application(
 
   def status =
     closed match {
-      case true                                                              => "Clôturée"
-      case _ if answers.filterNot(_.creatorUserID != creatorUserId).nonEmpty => "Répondu"
-      case _ if seenByUserIds.intersect(invitedUsers.keys.toList).nonEmpty   => "Consultée"
-      case _                                                                 => "Nouvelle"
+      case true                                                            => "Clôturée"
+      case _ if !answers.forall(_.creatorUserID != creatorUserId)          => "Répondu"
```

Comment:
y'a pas un `!==` avec `Eq` ?

```suggestion
      case _ if answers.exists(_.creatorUserID != creatorUserId)          => "Répondu"
```